mobile/src/main/res/xml/item_update_widget.xml

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="utf-8"?>
+<appwidget-provider xmlns:android="http://schemas.android.com/apk/res/android"
+    android:minWidth="40dp"
+    android:minHeight="40dp"
+    android:updatePeriodMillis="0"
```

Comment:
I set this to 0, but `onUpdate()` is called quite often. I should try setting a pretty high number here.